osf/management/commands/populate_branched_from_node.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import datetime
+
+from django.core.management.base import BaseCommand
+from framework.celery_tasks import app as celery_app
+from django.db import connection, transaction
+
+logger = logging.getLogger(__name__)
+
+POPULATE_BRANCHED_FROM_NODE = """UPDATE osf_abstractnode a
+ SET branched_from_node = CASE WHEN
+    EXISTS(SELECT id FROM osf_nodelog WHERE action='project_created_from_draft_reg' AND node_id = a.id) THEN False
+    ELSE True
+END
+WHERE
+a.type = 'osf.registration' AND
+a.branched_from_node IS null
```

Comment:
Do you need a LIMIT statement inside the sql in order to properly use the `page_size` parameter that you're passing in on L24? Something like `LIMIT %s`.

Reply:
Ah! Thanks!